### ISSUE TITLE
During a run Input Agents that enter after a script update use new script

### ISSUE THREAD

**Issue by benloh** [theRAPTLab/gsgo#96]
During a run, if a script is submitted, any CharControllers that introduce a new agent (either by coming online or by reloading) will automatically create an agent using the new updated script.  So you could potentially end up with two types of agents interacting.

This might be difficult to fix.

# To Test

0. Start `aquatic` with a few controllers.
1. Run the sim
2. Edit Fish Script: Add a `featCall Costume setGlow 0.1` to an `every 1` loop.
3. Submit the script.
4. The existing Fish should exhibit the old Fish behavior.
5. Reload one of the controllers -- The new fish will flash, but others do not.

Basically the old agents continue to use the old agent code while the new agents use the newly compiled blueprint code.  We'd have to do a lot of caching and managing to fix this and it's probably not worth the effort.
